lib/streak/search.rb

```diff
@@ -1,7 +1,8 @@
 module Streak
   class Search < StreakObject
     def self.query(s_query)
-      res = Streak.request(:get, "/v1/search", {query: s_query})
+      query = s_query.is_a?(String) ? {query: s_query} : s_query
```

Comment:
I don't like to do it that way. I think the guy has to send the right parameter. Except when it is something polymorphic. Not a big deal, just something personal.

Reply:
if you manage to adjust everything that calls query(s_query) to use the same format (string), great. if not, leave it that way.

Reply:
It has been updated!